staker/calculate_pool_position_reward.gno

```diff
@@ -518,3 +517,13 @@ func externalDepositGnsAmount() uint64 {
 
 	return amount
 }
+
+
+func getPoolTiers() map[string]InternalTier {
```

Comment:
should be avl

Reply:
calc_pool_position_XXX files will be removed. don't need to change